content/contributing/contributing-to-documentation.md

```diff
@@ -0,0 +1,166 @@
+| `fb_card`            | String           | Filename of an image (in `assets/img/`) to use when unfurling links to this page on Facebook. |
+| `twitter_card`       | String           | Filename of an image (in `assets/img/`) to use when unfurling links to this page on Twitter. |
+| `redirect_url`       | String           | Use with `template: pagetype-redirect.html.jinja` only. Automatically redirect the user to the provided URL when they navigate to this page. |
+| `cta_text`           | String           | Text to appear in "call to action" buttons that link to this page (on special landings). |
+| `curated_anchors`    | Array of Objects | A set of anchors in this page to show, similar to child pages, in landings. Each object in the array should have a human-readable `name` field (such as `"Available Modes"`) and an `anchor` field with the HTML ID to link to (such as `"#available-modes"`). |
+| `skip_spell_checker` | Boolean          | Use `true` to make the Dactyl's style checker skip spell-checking this page. |
+| `filters`            | Array of Strings | A list of additional filters to use on this page. [Filters](https://github.com/ripple/dactyl/blob/master/README.md#filters) are Python scripts that provide additional pre- or post-processing of page contents. |
+| `canonical_url`      | String           | Provides the canonical URL for a page that takes query parameters. Search engines and other tools may use this when linking to the page. |
+| `embed_xrpl_js`      | Boolean          | Use `true` to have the latest version of [xrpl.js](https://js.xrpl.org) loaded on the page. |
+
+### Conventions
+
+Use the following conventions when creating a page:
+
+- The HTML filename and MD filename should match exactly except for the file extension.
+- The filenames should closely match the title of the page, including words like "and" and "the", but should be in all lowercase with hyphens instead of spaces and punctuation. For example, `cash-a-check-for-an-exact-amount.md`. If you change the title of a page, change the filename too. (If it has already been published at another URL, leave a redirect from the old URL.)
+- Always start a page with a h1 header.
+- Don't link to the top h1 anchor of a page, link to the page itself without an anchor. This helps prevent broken links in translation. It's OK to link to later headers.
+- Don't use any formatting (like _italics_ or `code font`) in the title of the page.
+- Don't hard-wrap text in Markdown files.
+- For code samples, try to keep lines no longer than 80 columns wide.
+- When in doubt, follow [Ciro Santilli's Markdown Style Guide (Writability Profile)](https://cirosantilli.com/markdown-style-guide/).
+- Landing pages should be in subfolders and should have the same filename as the folder. For example, the landing page of the "Accounts" page group should be `accounts/accounts.md` with the HTML filename `accounts.html`. **Don't** use `index.md`.
```

Comment:
If this is serious enough to use bold on _Don't_, it might warrant a warning. Not sure why we're suddenly using bold text for emphasis this deep into the page. Personally, I'd avoid the bold so as not to pop people in the eye when we're discussing an entire list of do's and don'ts.

Reply:
We can remove the "don't use `index.md`" guideline anyway; we're going to need to use it for Redocly in the future.